Tasks/BelkevichAleksandr/Task1/task1.py

```diff
@@ -1,14 +1,13 @@
-a = 3
-b = -14
-c = -5
+deposit = int(input("Input your deposit: "))
+tern = int(input("Input deposit terns: ")) * 12
+persentage = int(input("Input annual percentage: ")) / 100
```

Comment:
also, this solution is related to money, so usage of the decimal data type is crucial, using int is unwise, -1 point for this